examples/losses_triplet.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>The clusters look great! Maybe showing the class for each cluster would be much better!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Good idea. It's a bit difficult to do in the projector. Let me know if the updated image works.</p>